Sources/BraintreeDataCollector/Kount/KDataCollector.h

```diff
@@ -86,6 +86,8 @@ typedef NS_ENUM(NSInteger, KEnvironment) {
 @property NSInteger timeoutInMS;
 // The Kount environment
 @property KEnvironment environment;
+// Device Data Sending to Analytics SDK
+@property NSDictionary *deviceDataForAnalytics;
```

Comment:
🤯  : Does the declaration of this property make a field in Kount's lib public? Or do we set it at some point.

Reply:
I don't think our SDK is using this field. It's just a new property that Kount added to their SDK.